tests/features/corporate-information.feature

```diff
@@ -0,0 +1,9 @@
+@api
+Feature: Corporate information
+  In order to be able to set and assert corporate information in my Behat tests
+  As a developer
+  I want to make sure that the steps provided by this module work correctly.
+
+  Scenario: Set and assert content owner.
+    Given I set the site owner to "Directorate-General for Informatics"
```

Comment:
This test would make more sense as a kernel test right? A behat test does not seem to offer any value to the PrOw. Unless we have this test as a showcase of the custom step, in which case its fine.

Reply:
Yeah it's only to test the steps themselves, as i say in the feature secription: As a developer...